pulsar-broker/src/main/java/org/apache/pulsar/broker/loadbalance/LinuxInfoUtils.java

```diff
@@ -254,7 +254,7 @@ public double convertBy(double usageBytes) {
 
     @AllArgsConstructor
     public enum UsageUnit {
-        Kbps(8 / 1024);
+        Kbps(8 / 1000);
```

Comment:
Nit: this enum's name was confusing to me. After digging through the code, I can see that this enum's role is to convert the host's bytes in/out into kilobits in/out for comparison against the kilobits/sec rate. It could be valuable to either rename this enum or add a comment to the enum for future readers of the code.

Reply:
@michaeljmarshall  It's my fault, I will change the enum name at another PR to make it easy to understand.

Reply:
No worries, and thanks @mattisonchao!